tests/suite/loading/yaml.typ

```diff
@@ -20,3 +20,22 @@
 // Warning: 15-21 `yaml.decode` is deprecated, directly pass bytes to `yaml` instead
 // Hint: 15-21 it will be removed in Typst 0.15.0
 #let _ = yaml.decode
+
+--- yaml-encode-any ---
+// Anything can be encoded.
+// Unsupported types fall back to strings via `repr`, but the specific output can be changed.
+#let check(value) = test(
+  yaml.encode(value),
+  yaml.encode(repr(value)),
+)
+
+#check(bytes("Typst"))
+#check(decimal("2.99792458"))
+#check(3.14159pt)
+#check(2.99em)
+#check(<sec:intro>)
+#check(panic)
+#check(x => x + 1)
+#check(regex("\p{Letter}"))
+#check(stroke(red + 5pt))
+#check(auto)
```

Comment:
since these are always the same (they are right?), it might make more sense to centralize this is in one test that tests all formats. or could you import the array from a common file, that might be even nicer (I think the test runner supports this).

Reply:
They are the same except that CBOR tests `bytes` in another place.

In fact at first, I centralize them together using a for loop, but I found that the error messages were ambiguous, so I manually separated them.

But if you insist, I can change it back.

Reply:
if the error message are a problem, you can leave it as-is